lib/elixir_console_web/live/console_live/sidebar_component.html.leex

```diff
@@ -35,7 +35,8 @@
         </p>
       <% end %>
     <% end %>
-    <ul>
+    <%# 'test-' prefixed class used only for test selectors %>
+    <ul class="test-suggestion-list">
```

Comment:
wdyt about not using the prefix as it can be confusing to see it when running the server? Also an id could be used to identify this tag instead of a class.

Reply:
Yeah, not super happy about this...

What I was trying to avoid was the possible confusion of coming back at this code in the future and by seeing `class="suggestion-list"`, expecting that to be a class used in a CSS file for styling purposes, but later on realizing not the case and being only used for a selector on a test case.

Similar in spirit to `js-` prefixed CSS classes when used only for selecting in JS files...

I can go with `id` instead though...

Reply:
Yeah I'd go with id="suggestion-list" but I'm ok with whatever you decide

Reply:
Cool - id works for me :-)